native/cocos/platform/android/java/src/com/cocos/lib/CocosSensorHandler.java

```diff
@@ -67,13 +69,18 @@ public CocosSensorHandler(final Context context) {
     // Getter & Setter
     // ===========================================================
     public void enable() {
-        mSensorManager.registerListener(this, mAcceleration, mSamplingPeriodUs);
-        mSensorManager.registerListener(this, mAccelerationIncludingGravity, mSamplingPeriodUs);
-        mSensorManager.registerListener(this, mGyroscope, mSamplingPeriodUs);
+        if (mEnableSensor) {
+            mHasRegisterListener = true;
```

Comment:
Why need mHasRegisterListener? It seems the same effect as mEnableSensor.

Reply:
App maybe crash, while disable sensor, but call unregisterListener. Need to be recheck.

Reply:
But it is only set to true here. And when it is set to true, no place to set it false.

Reply:
The recheck result is unregisterListener can be call multi-times. I have fix it